src/content/eth2/shard-chains/index.md

```diff
@@ -48,7 +48,11 @@ When the first shard chains are shipped they will just provide extra data to the
 
 Rollups are a "layer 2" technology that exists today. They allow dapps to bundle or “roll up” transactions into a single transaction off-chain, generate a cryptographic proof and then submit it to the chain. This reduces the data needed for a transaction. Combine this with all the extra data availability provided by shards and you get 100,000 transactions per second.
 
-[More on rollups](/developers/docs/layer-2-scaling/)
+<InfoBanner isWarning={false}>
+  Given recent progress in layer 2 scaling solution research and development, this has prompted the prioritization of the merge upgrade ahead of shard chains. These will be the focus following the mainnet transition to proof-of-stake.
```

Comment:
It's not clear what "these" is referring to in the second sentence. 

"Given the scalability improvements offered by layer 2 solutions, the merge upgrade will be prioritised ahead of shipping shard chains."

Reply:
still outstanding